cds-snc/notification-admin
# Change the problem email percentage scale from 2 to 1

# Summary | Résumé
This PR changes the number of digits after the decimal point of the problem email address percentage on both the dashboard and problem-emails pages.

# Testing
- Send some emails to `bounce@simulator.amazonses.com` and observe the problem email addresses block on the dashboard. It should now display only one digit after the decimal point in the percentage.
- Check the `problem-emails` page and check that the percentage also displays with one digit after the decimal point.
